tutorials/Coordinates/Coordinates.ipynb

```diff
@@ -0,0 +1,781 @@
+     "input": [
+      "import urllib\n",
+      "import IPython.display\n",
+      "import numpy as np"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "%matplotlib inline\n",
+      "from matplotlib import pyplot as plt"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "from astropy import units as u\n",
+      "from astropy.coordinates import SkyCoord\n",
+      "from astropy.table import Table"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "heading",
+     "level": 2,
+     "metadata": {},
+     "source": [
+      "Usinging Coordinates to describe on-sky locations and match catalogs"
+     ]
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Lets start by considering a field around the picturesque Hickson Compact Group 7.  To do anything with this, we need to get an object that represents the coordinates of the center of this group.\n",
+      "\n",
+      "In Astropy, the most common object you'll work with for coordinates is `SkyCoord`.  A `SkyCoord` can be created most easily directly from angles as shown below.  It's also wise to explicitly specify the frame your coordinates are in, although this is not strictly necessary because the default is ICRS. \n",
+      "\n",
+      "(If you're not sure what ICRS is, it's basically safe to think of it as an approximation to an equatorial system at the J2000 equinox)."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "hcg7_center = SkyCoord(9.81625*u.deg, 0.88806*u.deg, frame='icrs')\n",
+      "hcg7_center"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "SkyCoord will also accept string-formatted coordinates, although you'll need to give units if it's not obvious from the format."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "SkyCoord('0h39m15.9s', '0d53m17.016s', frame='icrs')"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "SkyCoord('0:39:15.9', '0:53:17.016', unit=(u.hour, u.deg), frame='icrs')"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "If the object you're interested in is in SIMBAD, you can also look it up directly from its name using the `SkyCoord.from_name()` [class method](https://julien.danjou.info/blog/2013/guide-python-static-class-abstract-methods). \n",
```

Comment:
The offsite link looks good.  But might be worth adding one sentence about what classmethods are for those who aren't familiar, and don't want to break the flow of the tutorial *right now* to read more about them.  Maybe something like "Think of this like an alternative constructor for a `SkyCoord` object--instead of calling `SkyCoord()` with some coordinates you use this to create a new `SkyCoord` given the name of some object."